contracts/governance/extensions/GovernorTimelockAccess.sol

```diff
@@ -97,17 +97,20 @@ abstract contract GovernorTimelockAccess is Governor {
      * delayed since queuing, and an array indicating which of the proposal actions will be executed indirectly through
      * the associated {AccessManager}.
      */
-    function proposalExecutionPlan(uint256 proposalId) public view returns (uint32, bool[] memory) {
+    function proposalExecutionPlan(
+        uint256 proposalId
+    ) public view returns (uint32 delay, bool[] memory indirect, bool[] memory withDelay) {
         ExecutionPlan storage plan = _executionPlan[proposalId];
 
-        uint32 delay = plan.delay;
         uint32 length = plan.length;
-        bool[] memory indirect = new bool[](length);
+        delay = plan.delay;
+        indirect = new bool[](length);
+        withDelay = new bool[](length);
         for (uint256 i = 0; i < length; ++i) {
-            (indirect[i], ) = _getManagerData(plan, i);
+            (indirect[i], withDelay[i], ) = _getManagerData(plan, i);
         }
 
-        return (delay, indirect);
+        return (delay, indirect, withDelay);
```

Comment:
I'm really struggling with this new convention... it's completely redundant to have this statement given the named return variables. I feel like I would rather remove it, or not have the named return variables.

Reply:
IMO, the named return variables are really usefull.

We have multiple option:
- redeclare the variable with a different name inside the body and do an explicit return. I'm not a fan of that because we need a different name for the same variable ...
- using the nameds return variable, and do an explicit return (like here). Its as explicit as one can be, but it can also be confusing. We actually don't need that last part
- remove the explicit return ... would that be more error prone ???

I tink we need to come up with guidelines for then entire repo.